mitosheet/src/jupyter/api.tsx

```diff
@@ -725,9 +725,32 @@ export default class MitoAPI {
         }, {})
     }
     
+    
+    async editBulkFilter(
+        sheet_index: number,
+        column_id: ColumnID,
+        toggle_type: 
+        | {'type': 'toggle_specific_value', 'value': string | number | boolean, 'remove_from_dataframe': boolean} 
+        | {'type': 'toggle_all_matching', 'search_string': string, 'remove_from_dataframe': boolean}
+    ): Promise<void> {
+
+        const stepID = getRandomId();
+        await this.send({
+            'event': 'edit_event',
+            'type': 'bulk_filter_edit',
+            'step_id': stepID,
+            'params': {
```

Comment:
I think we are trying to move to having this be 'params': {BulkFilterParams}. But no worries if not.